app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/CoinItem.kt

```diff
@@ -0,0 +1,157 @@
+package co.nimblehq.compose.crypto.ui.screens.home
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowDown
+import androidx.compose.material.icons.filled.KeyboardArrowUp
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.constraintlayout.compose.Dimension
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.constraintlayout.compose.ConstraintLayout
+import co.nimblehq.compose.crypto.R
+import co.nimblehq.compose.crypto.ui.theme.Color.FireOpal
+import co.nimblehq.compose.crypto.ui.theme.Color.GuppieGreen
+import co.nimblehq.compose.crypto.ui.theme.ComposeTheme
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp12
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp16
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp22
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp25
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp4
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp60
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp8
+import co.nimblehq.compose.crypto.ui.theme.Style
+import co.nimblehq.compose.crypto.ui.theme.Style.coinItemColor
+import co.nimblehq.compose.crypto.ui.theme.Style.coinNameColor
+import co.nimblehq.compose.crypto.ui.theme.Style.textColor
+
+@Suppress("FunctionNaming", "LongMethod")
+@Composable
+fun CoinItem(
+    isPositiveNumber: Boolean = false /* TODO Update value to Object on Integrate ticket */
+) {
+    ConstraintLayout(
+        modifier = Modifier
+            .wrapContentWidth()
+            .clip(RoundedCornerShape(Dp12))
+            .background(color = MaterialTheme.colors.coinItemColor)
```

Comment:
Why don't we set the padding 8 to root layout instead?

```suggestion
    ConstraintLayout(
        modifier = Modifier
            .wrapContentWidth()
            .clip(RoundedCornerShape(Dp12))
            .background(color = MaterialTheme.colors.coinItemColor)
            .padding(horizontal = Dp8, vertical = Dp8)
```